src/main/frontend/index.html

```diff
@@ -17,140 +17,115 @@
+        // Update all the siblings
+        let partial = 0;
+        siblings.forEach(function (slider, i) {
+            let val = +slider.value;
+            const fraction = sum > 0 ? val / sum : 1 / (sliders.length - 1);
+            if (i >= sliders.length - 1) {
+                val = max - partial;
+            } else {
+                val = Math.round(delta * fraction); // Round the value
+                partial += val;
+            }
+            set(slider, val);
+        });
+    }
 
-                // Set value on a slider
-                function set(elm, val) {
-                    if (val || val === 0) { // Check for both 0 and undefined
-                        elm.value = val;
-                    }
-                    elm.setAttribute('value', elm.value);
-                }
+    // Set value on a slider
+    function set(elm, val) {
+        if (val || val === 0) { // Check for both 0 and undefined
+            elm.value = val;
+        }
+        elm.setAttribute('value', elm.value);
+    }
 
-                // Add event listener to the specific sliders
-                const riverSlider = document.getElementById('riverSlider');
-                const artipSlider = document.getElementById('artipSlider');
-                const sugolSlider = document.getElementById('sugolSlider');
+    // Add event listener to the specific sliders
+    const riverSlider = document.getElementById('riverSlider');
+    const artipSlider = document.getElementById('artipSlider');
+    const sugolSlider = document.getElementById('sugolSlider');
+    const EHAMSlider = document.getElementById('EHAMSlider');
 
-                const sliders = [riverSlider, artipSlider, sugolSlider];
+    const sliders = [riverSlider, artipSlider, sugolSlider, EHAMSlider];
 
-                sliders.forEach(function (slider) {
-                    slider.addEventListener('input', function () {
-                        change(this);
-                    }, false);
-                });
+    sliders.forEach(function (slider) {
+        slider.addEventListener('input', function () {
+            change(this);
+        }, false);
+    });
 
-                // Function to send command
-                async function sendCommand() {
-                    let url = "http://127.0.0.1:5000/terminal/start"
-                    const response = await fetch(url);
-                    console.log(response)
-                }
-            </script>
-        </div>
-    </div>
+    // Function to send command
+    function downloadScene() {
+        let sliderval = [
+            riverSlider.value.toString(),
+            artipSlider.value.toString(),
+            sugolSlider.value.toString(),
+            EHAMSlider.value.toString()
+        ];
+        console.log(sliderval)
+        fetch(`http://127.0.0.1:5000/scene/scene?sortamount=${document.getElementById("sortamount").value}&slidervalue=${sliderval}&scenetime=${document.getElementById("scenetime").value}`,{
```

Comment:
env stuff. Also dont make this a one lines add some enters. I think there is a good plugin called Pretier that does this for you. It prop also exist in Jetbrains stuff